R/step-group.R

```diff
@@ -1,17 +1,30 @@
-step_group <- function(parent, groups = parent$groups) {
+step_group <- function(parent, groups = parent$groups, keyby = parent$keyby) {
   new_step(
     parent,
     vars = parent$vars,
     groups = groups,
-    class = "dtplyr_step_group"
+    class = "dtplyr_step_group",
+    keyby = keyby
   )
 }
 
+add_grouping_parameter <- function(call, groups, keyby) {
```

Comment:
Maybe just `add_grouping_param()`? And I notice that all the calls look like `add_grouping_parameter(something, x$groups, x$keyby)` so maybe the second argument should just be a step object?

Reply:
Done and done!